src/ui/main.cpp

```diff
@@ -101,7 +101,7 @@ int main(int argc, char *argv[])
 
     // There are no other instances: start a new server.
     a.startServer();
-    Editor::addEditorToBuffer();
+    //Editor::addEditorToBuffer();
```

Comment:
It no longer has any effect on performance and just eats more memory/increases startup time.